addOns/wappalyzer/src/main/java/org/zaproxy/zap/extension/wappalyzer/WappalyzerPassiveScanner.java

```diff
@@ -359,4 +361,11 @@ void setMode(Mode mode) {
     public boolean appliesToHistoryType(int historyType) {
         return PluginPassiveScanner.getDefaultHistoryTypes().contains(historyType);
     }
+
+    @Override
+    public void optionsChanged(OptionsParam optionsParam) {
+        final WappalyzerParam param = optionsParam.getParamSet(WappalyzerParam.class);
```

Comment:
This one could go too.

Reply:
Removed the final and actually just in-lined it to avoid the intermediate storage.